src/Exercise3.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const Button = (props) => (
+  <div className="flex justify-center items-center mt-16">
+    <button
+      className="bg-blue-500 text-white px-10 py-5 rounded-lg"
+      onClick={() => alert(props.text)}
+    >
+      Click me
+    </button>
+  </div>
+);
```

Comment:
To simplify you can destructure the text from the prop. And move the `Button` component into the component folder.